docs/docsite/rst/porting_guide_2.4.rst

```diff
@@ -0,0 +1,74 @@
+.. _porting_2.4_guide:
+
+*************************
+Ansible 2.3 Porting Guide
+*************************
+
+This section discusses the changes to how you use Ansible between Ansible 2.2 and Ansible 2.3.
+
+It is intended to assist in updating your playbooks, plugins, etc so they will work with this version of Ansible.
```

Comment:
Please avoid using "etc". Suggest changing to either simply "playbooks and plugins" or maybe "your Ansible infrastructure".